tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -1053,3 +1053,26 @@ def tearDown(self):
         self.apps = None
         self.data_layer = None
         MarionetteTestCase.tearDown(self)
+
+
+class PasscodeTestCase(GaiaTestCase):
+    def setPasscodeTo1337(self):
```

Comment:
Nit: Could you add a blank line before this, and rename the method to `set_passcode_to_1337`